server/apps/proxy-smtp/docker-compose.yml

```diff
@@ -0,0 +1,33 @@
+version: '3.0'
+
+# Explanation:
+# - starts a haproxy server on address haproxy:25 that forwards connections to james:2525
+# - the service "helo" just sends a HELO to haproxy that will be forwarded to james
+# - james receives the origin ip address of the helo container provided by haproxy
+# - you can also use telnet to connect to haproxy if you expose haproxy's port 25
+#
+# Caveat: james might take a while to start. Thus start james first,
+# then haproxy and finally helo. Observe the log ouput of the helo
+# container to see if it works correctly.
```

Comment:
I would be moving this explanation into a README.md file, which would make it easier to find and read using GitHub. WDYT ?

Reply:
Good idea.